charts/opensearch/values.yaml

```diff
@@ -31,71 +31,71 @@ global:
 # Allows you to add any config files in {{ .Values.opensearchHome }}/config
 opensearchHome: /usr/share/opensearch
 # such as opensearch.yml and log4j2.properties
-config:
-  # Values must be YAML literal style scalar / YAML multiline string.
-  # <filename>: |
-  #   <formatted-value(s)>
-  # log4j2.properties: |
-  #   status = error
-  #
-  #   appender.console.type = Console
-  #   appender.console.name = console
-  #   appender.console.layout.type = PatternLayout
-  #   appender.console.layout.pattern = [%d{ISO8601}][%-5p][%-25c{1.}] [%node_name]%marker %m%n
-  #
-  #   rootLogger.level = info
-  #   rootLogger.appenderRef.console.ref = console
-  opensearch.yml: |
-    cluster.name: opensearch-cluster
-
-    # Bind to all interfaces because we don't know what IP address Docker will assign to us.
-    network.host: 0.0.0.0
-
-    # Setting network.host to a non-loopback address enables the annoying bootstrap checks. "Single-node" mode disables them again.
-    # Implicitly done if ".singleNode" is set to "true".
-    # discovery.type: single-node
-
-    # Start OpenSearch Security Demo Configuration
-    # WARNING: revise all the lines below before you go into production
-    plugins:
```

Comment:
This file should be generated via the demo configuration script, if it is hard coded we will need to maintain this in two places.